composer.json

```diff
@@ -16,16 +16,16 @@
         }
     ],
     "require": {
-        "php": "^8.0.2",
-        "illuminate/support": "^9.0",
-        "illuminate/http": "^9.0",
-        "monolog/monolog": "^1.12|^2.0",
+        "php": "^8.1.2",
```

Comment:
Why did you change the PHP as well? I have a feeling this will break Laravel 9 compatibility.